src/sys/socket/mod.rs

```diff
@@ -274,10 +279,77 @@ impl<'a> Iterator for CmsgIterator<'a> {
 /// [Further reading](http://man7.org/linux/man-pages/man3/cmsg.3.html)
 pub enum ControlMessage<'a> {
     /// A message of type SCM_RIGHTS, containing an array of file
-    /// descriptors passed between processes. See the description in the
-    /// "Ancillary messages" section of the
+    /// descriptors passed between processes.
+    ///
+    /// See the description in the "Ancillary messages" section of the
     /// [unix(7) man page](http://man7.org/linux/man-pages/man7/unix.7.html).
     ScmRights(&'a [RawFd]),
+    /// A message of type SCM_TIMESTAMP, containing the time the packet
```

Comment:
Backticks around "SO_TIMESTAMP" here and on 290.

Reply:
The later usage of *"SO_TIMESTAMP"* is not referring to the identifier, it's referring to the section name, as is the earlier *"Ancillary messages"*.